src/mixins/LeadTime.js

```diff
@@ -0,0 +1,48 @@
+import { types } from 'mobx-state-tree';
+import debounce from  'lodash.debounce';
+
+const LEAD_TIME_CALCULATION_INTERVAL = 500;
+
+/**
+ * Calculate lead time just by calling `countTime()`. Result stored in `leadTime`.
+ */
+const LeadTimeMixin = types
+  .model({
+    leadTime: 0,
+  })
+  .volatile(() => ({
+    startTime: 0,
+  }))
+  .actions(self => ({
+    internalCountTime(callTime) {
+      const now = +Date.now();
```

Comment:
`Date.now()` returns a number, you don't need to convert it to number

Reply:
```suggestion
      const now = Date.now();
```